metricbeat/module/kafka/consumergroup/consumergroup.go

```diff
@@ -59,56 +54,38 @@ var debugf = logp.MakeDebug("kafka")
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	cfgwarn.Beta("The kafka consumergroup metricset is beta")
 
-	config := defaultConfig
-	if err := base.Module().UnpackConfig(&config); err != nil {
-		return nil, err
-	}
-
-	var tls *tls.Config
-	tlsCfg, err := tlscommon.LoadTLSConfig(config.TLS)
+	ms, err := kafka.NewMetricSet(base)
 	if err != nil {
 		return nil, err
 	}
-	if tlsCfg != nil {
-		tls = tlsCfg.BuildModuleConfig("")
-	}
 
-	timeout := base.Module().Config().Timeout
-
-	cfg := kafka.BrokerSettings{
-		MatchID:     true,
-		DialTimeout: timeout,
-		ReadTimeout: timeout,
-		ClientID:    config.ClientID,
-		Retries:     config.Retries,
-		Backoff:     config.Backoff,
-		TLS:         tls,
-		Username:    config.Username,
-		Password:    config.Password,
-
-		// consumer groups API requires at least 0.9.0.0
-		Version: kafka.Version("0.9.0.0"),
+	config := struct {
+		Groups []string `config:"groups"`
```

Comment:
Should we perhaps prefix groups with `consumergroup.group` as it seems to be a config only for this metricset?

Reply:
This setting already existed before, this would be a breaking change.

Reply:
Could we support both for 6.x, deprecate the old and remove it in 7?